### PR TITLE
Added search for country

### LINKED ISSUE #12: add search for country
Because of the long list of country, it could be great to have a search bar in the first element of the dropdown to filter the countries

### PR DESCRIPTION
**Pull request by pontes-guilherme**
closes #12

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -23,19 +23,39 @@
               aria-controls="dropdown-basic"> <span class="flag-icon flag-icon-{{selected.code.toLowerCase()}}"></span> {{ selected.name }} <span class="caret"></span>
       </button>
       <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu dropdown-menu-right scrollable-menu" role="menu" aria-labelledby="button-basic">
-        <li role="menuitem" *ngFor="let country of countryList" (click)="select(country)">
+        <li>
+          <div class=" px-4 py-2">
+            <input type="text" class="form-control" placeholder="Search your country..." [(ngModel)]="searchContent" (keyup)="searchCountry()" (click)="$event.stopPropagation();" />
```
Comment (arnaud512) [inline src/app/app.component.html:28]: Can you update the placeholder to "Search a country" and add a magnifier on the left?

File: src/app/app.component.html [hunk 2]
```diff
@@ -23,19 +23,39 @@
...
+          </div>
+        </li>
+        <li role="menuitem" *ngFor="let country of filteredCountries" (click)="select(country)">
           <a class="dropdown-item"><span class="flag-icon flag-icon-{{country.code.toLowerCase()}}"></span> {{country.name}}</a>
         </li>
+
+        <li role="menuitem" *ngIf="filteredCountries.length === 0">
+          <div class=" px-4 py-2">
+            <span>No items found</span>
```
Comment (arnaud512) [inline src/app/app.component.html:37]: Can you update this text to "No country found" ?

**Review by arnaud512 — CHANGES_REQUESTED** [review]
just a small text update and it will be ok!

File: src/app/app.component.html (hunk 1 above)
Comment (arnaud512) [inline src/app/app.component.html:28]: Can you update the placeholder to "Search a country" and add a magnifier on the left?

**Review by arnaud512 — CHANGES_REQUESTED** [review]
just a small text update and it will be ok!

File: src/app/app.component.html (hunk 2 above)
Comment (pontes-guilherme) [reply src/app/app.component.html]: For sure! I just committed the requested changes

File: src/app/app.component.html (hunk 1 above)
Comment (pontes-guilherme) [reply src/app/app.component.html]: This update was just committed as I mentioned above

**Comment by arnaud512** [comment]
hey @pontes-guilherme you have three files in conflicts!
Can you fix it? 
Btw, the changes are good enough for me! Thanks

**Comment by pontes-guilherme** [comment]
The last commit resolved the conflicts.
I'm glad I could contribute!

**Review by arnaud512 — APPROVED** [review]

**Review by arnaud512 — APPROVED** [review]

**Comment by arnaud512** [comment]
merged!

**Pull request merged**